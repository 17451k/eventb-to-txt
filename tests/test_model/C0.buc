<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<org.eventb.core.contextFile org.eventb.core.configuration="org.eventb.core.fwd" version="3">
<org.eventb.core.carrierSet name="'" org.eventb.core.comment="comment" org.eventb.core.identifier="set1"/>
<org.eventb.core.carrierSet name="(" org.eventb.core.identifier="set2"/>
<org.eventb.core.axiom name=")" org.eventb.core.comment="comment" org.eventb.core.label="axm1" org.eventb.core.predicate="cst1 ∈ set1"/>
<org.eventb.core.axiom name="*" org.eventb.core.comment="multiline&#10;comment" org.eventb.core.label="axm2" org.eventb.core.predicate="cst2 ∈ set2"/>
<org.eventb.core.axiom name="+" org.eventb.core.comment="comment" org.eventb.core.label="axm3" org.eventb.core.predicate="TRUE ≠ FALSE" org.eventb.core.theorem="true"/>
<org.eventb.core.constant name="," org.eventb.core.comment="comment" org.eventb.core.identifier="cst1"/>
<org.eventb.core.constant name="-" org.eventb.core.comment="multiline&#10;comment" org.eventb.core.identifier="cst2"/>
</org.eventb.core.contextFile>
