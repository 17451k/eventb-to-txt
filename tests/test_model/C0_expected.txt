context C0

sets
    set1 // comment
    set2

constants
    cst1 // comment
    cst2 // multiline comment

axioms
    @axm1
        cst1 ∈ set1 // comment
    @axm2
        cst2 ∈ set2 // multiline comment
    theorem @axm3
        TRUE ≠ FALSE // comment

end
